portfolio/src/main/java/com/google/sps/data/TaskPriority.java

```diff
@@ -0,0 +1,17 @@
+package com.google.sps.data;
+
+/** The TraskPriority class handles range checking, and allows retrieval of the priority value. */
+public final class TaskPriority {
+  private final int priority;
+
+  TaskPriority(int priority) {
+    if (priority >= 1 || priority <= 5) {
```

Comment:
This case is valid. I think you need to negate. And it'd be nice to add some unit test on this class.

Reply:
Done.